scripts/west_commands/build.py

```diff
@@ -58,6 +62,12 @@
   cmake_opt             Extra options to pass to CMake; implies -c
 '''
 
+def config_get(option, fallback):
```

Comment:
Shouldn't this be used for line 164:
`pristine = config.get('build', 'pristine', fallback='never')`

Reply:
Sure, I can convert all the existing users too.